lib/ReactViews/DataCatalog/DisplayGroupService.ts

```diff
@@ -0,0 +1,85 @@
+import { runInAction } from "mobx";
```

Comment:
Is splitting into `*Service` file a pattern you have used before? Personally, I have been writing them as loose (outer functions) inside the component class itself. I don't hold a strong opinion at this point though.

Reply:
I have split it into a service file because these funtions are imported into two different components. 
Yeah its a pattern i used all the time, but now I think the term might be used more by Angular devs... Should I rename?

Reply:
Sounds good, although I like `*Helper` better. Service sounds like some daemon or process to me, but probably that's just me?

Reply:
Yeah, I think I must have got 'Service' from using Angular so much. Helper is a better generic term I will change